scripts/commons

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
+# Fix hostname and configure hosts
 set_correct_hostname() {
-    # Fix hostname name
     domain=$(sudo yunohost domain list | sed -n '2p' | sed 's/  - //')
-    if [ -n "$domain" ] && [ ! "$(hostnamectl --static | grep '\.')" ]; then
+    if [ -n "$domain" ] && ! hostnamectl --static | grep -q '\.'; then
```

Comment:
Yes, you need more coffee :p. Because, with bash, the negation sign covers the whole pipe chain.

Example:
```
% ( true | false ) ; echo $?
1
% ( ! true | false ) ; echo $?
0
```